src/tests/modify.test.js

```diff
@@ -298,3 +313,90 @@ describe('modify() - basic mutations', () => {
     });
   });
 });
+
+describe('modify() - set new fields', () => {
+  it('add base field', () => {
+    const result = modify(schemaAddress, {
+      add: {
```

Comment:
note: Regarding the "add" name I also thought about:
- `addField`, but we already have the other shorter names like `pick`, `omit` and `order`. So `add` seems better.
- `set`, but it's not "set" really, because "set" implies replacing existing fields, which does not do.
- `insert`, does not make sense because it implies a specific position, which in this case is not guaranteed (that's up to `order`)

What are your thoughts?

Reply:
Update: Eureka moment while writing the other comments, I'll rename this to `create` :D

Reply:
ahah `create` sounds good! Also thought of `inject` since you're not only creating the field property but "injecting" the field and it's properties into an existing schema

Reply:
we are not doing rocket science, no need to inject 🚀 ahahah

Reply:
@sandrina-p just letting you know that the PR description is not updated with `create`

Reply:
@eng-almeida updated :)